### PR TITLE
[feat]: Navbar

### LINKED ISSUE #2: remove unwanted lines of code.

### PR DESCRIPTION
**Pull request by saadahmadhere**
image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-	<body>
-		<noscript>You need to enable JavaScript to run this app.</noscript>
-		<div id="root"></div>
-	</body>
+    <link rel="stylesheet" href="https://chroma-ui.netlify.app/style.css" />
+
+    <title>Elmart</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (Hamdrive) [inline public/index.html]: You can cleanup this line as it is irrelevant

File: src/Components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { Link } from "react-router-dom";
+import "./Navbar.css";
+
+const Navbar = () => {
+  return (
+    <nav className="navbar flex">
+      <div className>
```
Comment (Hamdrive) [inline src/Components/Navbar/Navbar.jsx]: You have left an empty `className` attribute, might have forgotten to add the classes, else if not in use then cleanup

**Review by Hamdrive — COMMENTED** [review]
Good work done, code looks clean and good React coding practices can be see. 
Have suggested some small fixes which occur during porting from HTML to JSX

**Review by shivampandey0 — APPROVED** [review]
Standard coding practices followed.
Good to merge.

File: src/Components/Navbar/Navbar.jsx (hunk 2 above)
Comment (saadahmadhere) [reply src/Components/Navbar/Navbar.jsx]: done! Thanks for pointing out.

File: public/index.html (hunk 1 above)
Comment (saadahmadhere) [reply public/index.html]: done!

**Pull request merged**